packages/react-next/README.md

```diff
@@ -0,0 +1,16 @@
+# [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)
+
+**The React-based front-end framework for building experiences for Microsoft 365.**
+
+**Fluent UI 8**
+
+This package represents a staging ground for breaking changes in v8. We will be
+staging potentially breaking updates here with the intent that
```

Comment:
unfinished sentence (also please combine the lines rather than manual line breaks)

Reply:
And add a note about API stability or lack thereof